pycbc/waveform/bank.py

```diff
@@ -307,6 +310,12 @@ def __init__(self, filename, approximant=None, parameters=None,
         self.extra_args = kwds
         self.ensure_hash()
 
+        # Variables for the injection-segment filter checker
```

Comment:
Can you move the logic to generate your "short" cached template to a method of the FilterBank class? I'm thinking this is more generically useful for also doing a "short" sigmasq estimate.